lib/context/AuthoritiesSearchContext.js

```diff
@@ -63,17 +59,11 @@ const AuthoritiesSearchContextProvider = ({
     option: searchParams.qindex,
   });
 
-  const setNavigationSegmentValue = value => {
```

Comment:
Please check that bug in https://github.com/folio-org/stripes-authority-components/pull/15 is still fixed

Reply:
Yes, it is still fixed.

https://user-images.githubusercontent.com/77053927/206396537-94f7180b-5a53-49c4-80a1-ab6d80f73737.mp4